src/main/_docs/use-che-as-an-ide/ide-ssh.md

```diff
@@ -20,7 +20,10 @@ We then provide various client utilities (authored in Docker!) for connecting to
 # List Workspaces  
 You can get a list of workspaces in a Che server that have an SSH agent deployed. These are the workspaces that you can SSH into.
 ```shell  
-$ che action list-workspaces
+$ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Yeah, was thinking of this - but then I was asking myself, if I'm discovering the product, I would prefer to get whole instructions, otherwise I would forgot what are those and would have to look for them into another page.

It's less readable - but more useful this way.